src/uct/tcp/tcp_sockcm_ep.h

```diff
@@ -21,7 +21,8 @@ typedef enum uct_tcp_sockcm_ep_state {
     UCT_TCP_SOCKCM_EP_CLIENT_NOTIFY_SENT          = UCS_BIT(10), /* ep on the client sent the notify message to the server */
     UCT_TCP_SOCKCM_EP_DISCONNECTING               = UCS_BIT(11), /* @ref uct_ep_disconnect was called on the ep */
     UCT_TCP_SOCKCM_EP_GOT_DISCONNECT              = UCS_BIT(12), /* ep received a disconnect notice from the remote peer */
-    UCT_TCP_SOCKCM_EP_FAILED                      = UCS_BIT(13)  /* ep is in error state due to an internal local error */
+    UCT_TCP_SOCKCM_EP_FAILED                      = UCS_BIT(13), /* ep is in error state due to an internal local error */
+    UCT_TCP_SOCKCM_EP_CLIENT_GOT_REJECTED         = UCS_BIT(14), /* ep on the client side received a reject from the server */
```

Comment:
why need another state? can't we reuse UCT_TCP_SOCKCM_EP_GOT_DISCONNECT?

Reply:
we can but isn't it better for debug?

Reply:
so we can set it in addition, but not let it affect the decision making